WindowsStubs/WindowsStubsTest/WindowsStubsTest.vcxproj

```diff
@@ -110,7 +110,7 @@
       <SubSystem>Console</SubSystem>
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <AdditionalDependencies>WindowsStubs.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalLibraryDirectories>$(ProjectDir);$(ProjectDir)..\$(Configuration);$(ProjectDir)..\..\WindowsStubs\$(Configuration)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories>$(ProjectDir);$(ProjectDir)..\$(Configuration);$(ProjectDir)..\..\WindowsStubs\x64\$(Configuration)</AdditionalLibraryDirectories>
     </Link>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
```

Comment:
I don't understand why you committed this. I think you should delete this whole project.